### PR TITLE
New movie MiddleMen

### PR DESCRIPTION
**Pull request by learningsam20**
Adding new movie middlemen

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Awesome IT-films
+# Awesome IT-films <img src="/assets/ITMovieHub.png" width="200">
```
Comment (greybax) [inline README.md]: Please add this logo in another PR

Comment (greybax) [reply README.md]: And remove also file from assets.. Please send this change in another PR

**Comment by greybax** [comment]
@learningsam20 please fix comments in PR and after that I’ll merge your PR

**Comment by learningsam20** [comment]
Done

**Comment by greybax** [comment]
Merged

**Pull request merged**